src/persist-cli/src/maelstrom/txn_list_append_multi.rs

```diff
@@ -112,12 +114,15 @@ impl Transactor {
                     txn.write(&data_id, data, (), diff).await;
                 }
             }
-            let mut write_ts = self.oracle.write_ts();
+            // TODO: Figure out how to stay linearizable while getting this
+            // write_ts from the oracle instead.
+            let mut write_ts = read_ts + 1;
```

Comment:
Maybe something like

```Rust
let mut read_ts;
let mut reads;

if writes.is_empty() {
  read_ts = self.oracle.read_ts().await?;
  reads = self.read_at(read_ts, read_ids.iter()).await;
else {
  let write_ts = self.oracle.write_ts().await?;
  read_ts = write_ts - 1;
  reads = self.read_at(read_ts, read_ids.iter()).await;
  ...
}
```

Reply:
Yeah, I wanted to do this too, but sadly found out the hard way that this and your other comment don't work :(

In particular, getting a `write_ts` doesn't guarantee that `write_ts-1` is readable. Someone else might have grabbed the timestamp before but crashed before using it